CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+reedline's development is currently primarily driven by the [nushell project](https://github.com/nushell) to provide its indigenous interactive REPL and explore options for pleasant interaction with a shell and programming language.
```

Comment:
remove "indigenous" and starting with "interactive REPL" reads a bit better.

Reply:
Like?
```
reedline's development is primarily driven by the [nushell project](https://github.com/nushell) at the moment to provide its interactive REPL and explore options for pleasant interaction with a shell and programming language.
```

Reply:
I split that wordy sentence :D